vmss-prototype/vmss-prototype

```diff
@@ -1048,7 +1048,7 @@ def vmss_prototype_update(sub_args):
             # provisioning bits on it (prototype) we can remove all extensions
             # But...  We want to keep the AKS Engine-identifying extension for telemetry reasons
             # It is a no-op code wise but gives counts
-            if 'vmss-computeAksLinuxBilling' not in extension['name']:
+            if 'AKSLinuxBilling' not in extension['name']:
```

Comment:
I should really add in what we did in Skyman that lets us provide a list of possible name patterns to exclude from the delete.  This allows us to experiment with certain conditions.  But this is good for what you are playing with.

PS - We actually do not exclude any extensions right now but that is just to reduce the number of things that happen at scale in to its absolute minimum.